app/modules/crystal.py

```diff
@@ -0,0 +1,19 @@
+import app.module
+
+import math
+
+
+class CrystalModule(app.module.Module):
+    def __init__(self, client):
+        app.module.Module.__init__(self, client, "crystal", self.calculate)
+
+    async def calculate(self, rawMessage, tokens):
```

Comment:
Why use an invalid command handler to calculate crystals? Ideally an invalid command handler would just give the user some sort of feedback that their command doesn't work imo.

I think it would be better if this was handled by some sort of "default" handler, so that when no specific commands are passed it will run the default process. Whether the way to do that is to just rename `invalidCommandHandler` to `defaultCommandHandler` or making an entirely separate default handler is up to you.

Reply:
Thank you for the feedback. I totally agree. Unfortunately, I blanked on the word "default" when I was writing this 🤷. I have renamed `invalidCommandHandler` to `defaultMessageHandler` in b27df0c. Please resolve this conversation if that is sufficient.